v4/export/dump.go

```diff
@@ -13,6 +13,9 @@ import (
 )
 
 func Dump(conf *Config) (err error) {
+	if !conf.EscapeBackslash {
+		globalEscape = &noBackslashEscape{}
```

Comment:
put it into config,init or config.adjust? same with L31

Reply:
addressed in https://github.com/pingcap/dumpling/pull/49/commits/e920a47b8342631c4838ef05b2185572e7d754dd